V5 Release note - Quality part (#873)
akeneo/pim-helpcenter: content/updates/quality/3-Spell-check-improvements.md

# Spell-check improvements

## On text area attributes with rich text editor enabled
::: meta-data type="Improvement" ee-only="true" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
:::

To help reach higher data quality content, spell-check and suggestions are now available on text area attributes with rich text editor enabled (WYSIWYG).

## On attribute and option labels
::: meta-data type="New" ee-only="true" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
:::

> Now, you can also benefit from spell-checking on attributes and option labels to guarantee the consistency on your customer-facing product data.

Reviewer: ```suggestion
Now, you can also benefit from spell-checking on attributes and option labels to guarantee the consistency of your customer-facing product data.
```

> You can find this improvement under the attribute and attribute option setting pages. It is also available in the product form. New recommendations are listed, should we find errors. 

Reviewer: under? I believe it should be ON?

Author: Not sure I understand the difference between the two meanings.

Reviewer: same here :)

Author: Fixed!

You can find this improvement under the attribute and attribute option setting pages. It is also available in the product form. New recommendations are listed, should we find errors. 

![dqi-tab-consistency-in-structure](../img/data-quality-spelling-option.png)

The better the content, the higher the quality score! :wink:

## On new languages supported
::: meta-data type="Improvement" ee-only="true"
:::

The spell-check feature now supports 7 new languages: Italian 🇮🇹, Swedish 🇸🇪, Danish 🇩🇰, Dutch 🇳🇱, Norwegian Bokmål 🇳🇴, Norwegian Nynorsk 🇳🇴 , and Brazilian Portuguese 🇧🇷 bringing the total number of supported languages to 11! With spelling mistakes automatically detected for so many locales, you will ensure data quality for most of your content! :wink:

## For single locale PIM
::: meta-data type="Improvement" ee-only="true"
:::

For those of you who have a catalog with only one locale enabled, we've got some great news for you. You can now benefit from our spell-checking feature for all your text and text area attributes if your locale is one of the 11 supported languages.

> 
> 

Author: ```suggestion
```

# Spell-check improvements

## On text area attributes with rich text editor enabled
::: meta-data type="Improvement" ee-only="true" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
:::

To help reach a higher data quality content yet, spell-check and suggestions are now available on text area attributes with rich text editor enabled (WYSIWYG).

## On attribute and option labels
::: meta-data type="New" ee-only="true" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
:::

Now, you can also benefit from spell-checking on attributes and option labels to guarantee the consistency on your customer-facing product data.

You can find this improvement under the attribute and attribute option setting pages. It is also available in the product form. New recommendations are listed, should we find errors. 

![dqi-tab-consistency-in-structure](../img/data-quality-spelling-option.png)

The better the content, the higher the quality score! :wink:

## On new languages supported
::: meta-data type="Improvement" ee-only="true"
:::

> The spell-check feature now supports 7 new languages: Italian 🇮🇹, Swedish 🇸🇪, Danish 🇩🇰, Dutch 🇳🇱, Norwegian Bokmål 🇳🇴, Norwegian Nynorsk 🇳🇴 and Brazilian Portuguese 🇧🇷. This means that you can now have spelling mistakes automatically detected for all those new locales. Mistakes are no longer allowed! :wink:

Author: ```suggestion
The spell-check feature now supports 7 new languages: Italian 🇮🇹, Swedish 🇸🇪, Danish 🇩🇰, Dutch 🇳🇱, Norwegian Bokmål 🇳🇴, Norwegian Nynorsk 🇳🇴 , and Brazilian Portuguese 🇧🇷 bringing the total number of supported languages to 11! With spelling mistakes automatically detected for so many locales, you will ensure data quality for most of your content! :wink:
```
